doc/kokkos-promotion.txt

```diff
@@ -0,0 +1,240 @@
+Summary:
+
+- Step 1: Testing Kokkos + KokkosKernels using test_all_sandia
+
+- Step 2: Testing of Kokkos + KokkosKernels integrated into Trilinos (scripts/trilinos-integration/*.sh)
+
+- Step 3: Close all issues labeled "InDevelop"
+
+- Step 4: Locally update CHANGELOG, merge into master, edit scripts/master_history.txt
+
+- Step 5: Locally snapshot new master of kokkos and kokkos-kernels into corresponding Trilinos branch (develop or temporary), issue PR to Trilinos
+
+- Step 6: Push local Kokkos master to GitHub (need Owner approval). Push local KokkosKernels master to GitHub (need Owner approval)
+
+Steps 1, 2, and 4 include testing that may fail. These failures must be fixed either by pull requests to Kokkos and/or KokkosKernels develop, or by creating a new Trilinos branch for parts of Trilinos that must be updated. This is what usually takes the most time.
+
+
+// -------------------------------------------------------------------------------- //
+
+
+Step 1: The following should be repeated on enough machines to cover all
+supported compilers. Those machines are:
+
+    kokkos-dev
+    blake
+    white
+    bowman
+    waterman
+
+  1.1. Clone kokkos develop branch (or just switch to it)
+
+         git clone -b develop git@github.com:kokkos/kokkos.git
+         cd kokkos
+
+  1.2. Create a testing directory
+
+         mkdir testing
+         cd testing
+
+  1.3. Run the test_all_sandia script with no options to test all compilers
+
+         nohup ../scripts/test_all_sandia &
+         tail -f nohup.out                   # to watch progress
+
+  1.4. Clone kokkos-kernels develop branch (or just switch to it)
+
+         git clone -b develop git@github.com:kokkos/kokkos-kernels.git
+         cd kokkos-kernels
+
+  1.4. Create a testing directory
+
+         mkdir testing
+         cd testing
+
+  1.5. Run the test_all_sandia script with no options to test all compilers
+
+         nohup ../scripts/test_all_sandia &
+         tail -f nohup.out                   # to watch progress
+
```

Comment:
Could you give the instruction to delete jobs when the test is failed ? You once told me but I do not remember it.

Reply:
Good point, 
`ctrl+z`
`kill -9 %1`

I'll update and add this.